scripts/poll_spi_result

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Wait for the results of an SPI execution.
+# Receives the image_unique_id of the execution.
+# Exits with:
+# - 0 if the test passed.
+# - 1 if the test failed.
+# - 2 if it timed out checking for results.
+
+readonly TIMEOUT=7200  # 2 hours.
```

Comment:
Yes, you are right with separating the jobs. Anyway we need it because we are currently collecting results from the only test that runs. But when we add the tests for updates and rollbacks we'll get more than one result in the json.
I'll merge this as a first step, and then split to make it cleaner.

Thanks Paul!